src/SWP/Bundle/CoreBundle/Controller/ContainerController.php

```diff
@@ -95,6 +96,39 @@ public function getAction(Request $request, $id)
     }
 
     /**
+     * Render single container and it's widgets.
+     *
+     * @ApiDoc(
+     *     resource=true,
+     *     description="Render single container and it's widgets.",
+     *     statusCodes={
+     *         200="Returned on success.",
+     *         404="Container not found"
+     *     }
+     * )
+     * @Route("/api/{version}/templates/containers/{id}/render", requirements={"id"="\d+"}, options={"expose"=true}, defaults={"version"="v1"}, name="swp_api_templates_render_container")
```

Comment:
okay, it is just a matter of adding a config to fosrestbundle afaik to support this, but since we don't extend fos rest controller it will need some more work I guess